README.adoc

```diff
@@ -98,7 +98,7 @@ repository. To start a discussion, prefer https://discord.gg/4R6RGFf[Discord]
 over GitHub issues.
 
 *Read the
-https://github.com/keep-network/tbtc/blob/master/CONTRIBUTING.md[Contributing
+https://github.com/keep-network/tbtc/blob/main/CONTRIBUTING.md[Contributing
```

Comment:
Could we make this a relative link rather than referencing a branch?

Reply:
Done, see c5e10dd4c35b34d8d0df8f5572e4138567fff802.